src/commands/configuration-management/module.ts

```diff
@@ -24,9 +24,15 @@ class Module extends IModule {
 
         configCommand.command("export")
             .description("Command to export package configs")
-            .requiredOption("--packageKeys <packageKeys...>", "Keys of packages to export")
+            .option("--packageKeys <packageKeys...>", "Keys of packages to export. Exports the latest deployed version only")
+            .option("--keysByVersion <keysByVersion...>", "Keys of packages to export by version")
             .option("--withDependencies", "Include variables and dependencies", "")
-            .action(this.batchExportPackages);
+            .action(async (context: Context, command: Command, options: OptionValues): Promise<void> => {
+                if ((options.packageKeys && options.keysByVersion) || (!options.packageKeys && !options.keysByVersion)) {
+                    throw new Error("Please provide either --packageKeys or --keysByVersion, but not both.");
+                }
```

Comment:
I think it's better if we keep the action callback clean here, and implement the validation on the `batchExportPackages` method.

Reply:
Changed it [here](https://github.com/celonis/content-cli/pull/239/commits/f04b5b6c60be7cc2af7129ba4cf6061358e7459a).